include/seqan3/search/fm_index/bi_fm_index_cursor.hpp

```diff
@@ -437,6 +437,41 @@ class bi_fm_index_cursor
         return false;
     }
 
+    //!\overload
+    template <typename char_t>
+    bool extend_right(char_t* const c) noexcept
```

Comment:
can you try if this works with `std::string_view` instead?

Reply:
Actually, for `std::string_view` the range overload of `extend_right` works; so we could also do
```cpp
    bool extend_right(char const * c) noexcept
    {
        return extend_right(std::string_view{c});
    }
```
And be done with it (because the actual problem is the `\0` of the input)

Reply:
Oh you can do this?

The wouldn't be

```cpp
    bool extend_right(std::string_view sv) noexcept
    {
        return extend_right(sv);
    }
```

possible?